rental.rb

```diff
@@ -0,0 +1,9 @@
+class Rental
+  attr_accessor :date, :person, :book
+
+  def initialize(date, person, book)
+    @date = date
+    @person = person
```

Comment:
-  [ ] The rental should add itself to the @person the 'belongs-to' association this is how it would look like like 👇 
```ruby
person.rentals << self
```

Reply:
@hiromataba Thanks for the feedback, I really appreciate the time you take to teach me this.